landlab/components/flow_dir_acc_pf/cfuncs.pyx

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+cimport numpy as np
+cimport cython
+
+
+DTYPE_INT = np.int
+ctypedef np.int_t DTYPE_INT_t
+
+DTYPE_FLOAT = np.double
+ctypedef np.double_t DTYPE_FLOAT_t
+
+
+cpdef _FD_D8(np.ndarray[DTYPE_INT_t, ndim=1] receivers,
+             np.ndarray[DTYPE_FLOAT_t, ndim=1] distance_receiver,
+             np.ndarray[DTYPE_FLOAT_t, ndim=1] steepest_slope,
+             np.ndarray[DTYPE_FLOAT_t, ndim=1] el,
+             np.ndarray[DTYPE_FLOAT_t, ndim=1] el_ori,
+             np.ndarray[DTYPE_FLOAT_t, ndim=1] dist,
+             np.ndarray[DTYPE_INT_t, ndim=1] ngb,
+             np.ndarray[DTYPE_INT_t, ndim=1] cores,
+             np.ndarray[DTYPE_INT_t, ndim=1] activeCells,
+             np.ndarray[DTYPE_FLOAT_t, ndim=1] el_d,
+             DTYPE_INT_t r, DTYPE_INT_t c,
+             DTYPE_FLOAT_t dx, DTYPE_FLOAT_t sq2,
+             np.ndarray[DTYPE_INT_t, ndim=2] adj_link ,
+             np.ndarray[DTYPE_INT_t, ndim=1] rec_link):
+    
+    """
+    Calcualte D8 flow dirs
+    """
+    cdef int idx, i
+    
+    # for i in range(0,r*c):      
+    for i in cores:             
+        if activeCells[i]:  
```

Comment:
solved by passing activeCores where: 
activeCores = cores[activeCells[cores]==1]

Reply:
Remember to use *snake_case* for variables names and `NodeStatus.CORE` instead of *1*.

```python
active_cores = cores[active_cells[cores]] == NodeStatus.CORE]
```